app/uk/gov/hmrc/gform/validation/ValidationService.scala

```diff
@@ -496,7 +485,7 @@ class ComponentsValidator(
     xs: Seq[String]): ValidatedType =
     xs.filterNot(_.isEmpty()) match {
       case Nil =>
-        Map(fieldId -> errors(fieldValue, s"${errorPrefix.getOrElse(messagePrefix(fieldValue))} must be entered")).invalid
+        Map(fieldId -> errors(fieldValue, s"${errorPrefix.getOrElse(" ")}must be entered")).invalid
```

Comment:
```
${errorPrefix.getOrElse(" ")}must be entered
```
may be replaced by:
```
${errorPrefix.getOrElse("")} must be entered
```
And on call side we can drop extra spaces

Reply:
fixed with latest commit